content/workshops/ansible_tower/wrapup.adoc

```diff
@@ -0,0 +1,32 @@
+---
+title: "Wrapup"
+workshops: ansible_tower
+workshop_weight: 70
+layout: lab
+---
+
+:badges:
+:icons: font
+:iconsdir: http://people.redhat.com/~jduncan/images/icons
+:imagesdir: /workshops/ansible_tower/images
+:source-highlighter: highlight.js
+:source-language: yaml
+
+:figure-caption!:
+
+= Wrap Up and Q&A
+
+That wraps up what we have planned for today.  We hope you've learned something valuable +
+about Ansible and Ansible Tower that you can apply in your daily role.
+
+What do you think? How can we help you understand Ansible Tower better?
+
+Before you leave, check out the *Resources* page that is part of this guide.  There you can +
+find a ton of links that will further your Ansible education.
+
+This *Participant Guide* will remain active for the next two weeks.  Please take advantage +
+of it by creating your own Ansible and Ansible Tower environment back at your organization.
+
+Thank you for your time and participation!
+
+image::redhat.svg[]
```

Comment:
Don't put the Red Hat logo at the bottom of each of your lab pages. We can make this part of the template if you'd like so that it's automatically done.